openedx/core/djangoapps/credentials/tests/mixins.py

```diff
@@ -38,14 +39,14 @@ class CredentialsDataMixin(object):
                 id=1,
                 username='test',
                 credential=factories.ProgramCredential(
-                    program_id=1
+                    program_uuid=str(uuid.uuid4())
```

Comment:
Do you need to specify this? If left out, the factory should populate it automatically.